tiledb/sm/cpp_api/array.h

```diff
@@ -650,9 +654,19 @@ class Array {
 
   // clang-format off
   /**
-   * @copybrief Array::consolidate(const Context&,const std::string&,tiledb_encryption_type_t,const void*,uint32_t,const Config&)
+   * @copybrief Array::consolidate(
```

Comment:
I think `@copybrief` needs to be on a single line or doxygen will break (that's why `clang-format off` is above). The docs may not be rendered properly with this change currently. Same with the `@ref`s I believe

Reply:
oh I see, I will fix this.